src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -107,7 +98,7 @@ const EnvironmentPicker = ({
         Boolean(isDialogHiddenProp)
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236272-4dcf55b7-358b-4891-b761-d8a183c02be6.png)

Reply:
had commented and marked as "resolved" this on the original PR